### PR TITLE
Feat/make rta url params optional

### PR DESCRIPTION
**Pull request by heimwege**
This PR makes the usage of url params for the rta editor endpoints optional. In case they are missing a re-route will happen and the parameters will be added 🥳

### CODE REVIEW

**Comment by heimwege** [comment]
@ytpo-lyne this PR should help us for https://github.com/SAP/open-ux-tools/tree/feat/create-variantsConfig because the url parameter for the rta editor endpoints are no longer needed with this PR.
@tobiasqueck or do we need to keep them because of some downward compatibility? But if this PR is merged first we should hopefully be fine 🤔

**Comment by heimwege** [comment]
@tobiasqueck I think we do not need to insist on this logic working with the connect API used by the karma test runner as rta editor is not subject to karma tests. Would you agree?

File: packages/preview-middleware/test/unit/base/flp.test.ts [hunk 1]
```diff
@@ -336,7 +341,7 @@ describe('FlpSandbox', () => {
 
         test('rta with developerMode=true and plugin', async () => {
             await server.get('/with/plugin.html').expect(200);
-            const response = await server.get('/with/plugin.html.inner.html').expect(200);
+            const response = await server.get('/with/plugin.html.inner.html').expect(302);
```
Comment (tobiasqueck) [inline packages/preview-middleware/test/unit/base/flp.test.ts:344]: We control exactly what is called as `inner.html` therefore, we should be able to also add the url parameters if required, thus, I would not expect that there is a redirect necessary for the inner html.
Is the test case just wrong i.e. should we test here with parameters or is there an unnecessary redirect now?

**Review by tobiasqueck — CHANGES_REQUESTED** [review]
See inline comment - the redirects for tests with `developerMode=true` are not expected.

File: packages/preview-middleware/test/unit/base/flp.test.ts (hunk 1 above)
Comment (heimwege) [reply packages/preview-middleware/test/unit/base/flp.test.ts:344]: Are the URL parameters needed at all in case of `inner.html`?

Comment (tobiasqueck) [reply packages/preview-middleware/test/unit/base/flp.test.ts]: Thanks for adjusting the test case

**Review by tobiasqueck — DISMISSED** [review]
* changeset ok
* code change is simple BUT very useful since it allows us to simplify the generated call scripts
* didn't test locally but reviewing the jest tests gives me enough confidence

**Review by tobiasqueck — DISMISSED** [review]

**Review by tobiasqueck — APPROVED** [review]

**Review by ytpo-lyne — APPROVED** [review]
- changeset present
- code changes are straight forward
- code coverage is excellent 
- tested locally, no issues found

**Pull request merged**